OICI: 211 filtre prescripteur

Iandraina-etech opened the pull request:
Switched the location module dependency to use the 'feature-add-prescriber-filter' branch, enabling new functionality related to prescriber filtering.

Iandraina-etech committed:
Update location module to new feature branch

Switched the location module dependency to use the 'feature-add-prescriber-filter' branch, enabling new functionality related to prescriber filtering.

kevel-dev commented:
Il ne s'agit pas de créer la branche add-prescriper-filter mais de
- créer la branche test-comores dans le module de location contenant les changes actuel y compris les prescipteurs car c'est sur l'instance de test déja. vous pouvez créer cette branche a partir du develop arkeup
- créer la branche main-comores dans le module de location contenant tout les developpements de la production sans les prescripteurs.
- Apres ca il faut faire le nouveau PR vers be_py : test-comores aura dans son openimis.json la branche test-comores du module location.
- main-comores aura dans son openimis.json la branche main-comores du module location.

Iandraina-etech committed:
Update location module to use test-comores branch

Changed the 'location' module pip source to point to the 'test-comores' branch instead of 'feature-add-prescriber-filter'. This may be for testing or development purposes.
